### PR TITLE
Update make-circular-images-with-a-border-radius.russian.md

### PR DESCRIPTION
**Pull request by DmitriiKholodov**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md [hunk 1]
```diff
@@ -3,24 +3,24 @@ id: bad87fee1348bd9aedf08815
...
 <section id='tests'>
 
 ```yml
 tests:
-  - text: 'Ваше изображение должно иметь радиус границы <code>50%</code> , что делает его идеально круговым.'
-    testString: 'assert(parseInt($("img").css("border-top-left-radius")) > 48, "Your image should have a border radius of <code>50%</code>, making it perfectly circular.");'
-  - text: Обязательно используйте процентное значение <code>50%</code> .
-    testString: 'assert(code.match(/50%/g), "Be sure to use a percentage value of <code>50%</code>.");'
+  - text: 'Ваше изображение должно иметь радиус границы <code>50%</code> , что делает его идеально круглым.'
+    testString: 'assert(parseInt($("img").css("border-top-left-radius")) > 48, "Ваше изображение должно иметь радиус границы <code>50%</code> , что делает его идеально круглым.");'
```
Comment (greggubarev) [inline curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md]: Please turn back value of TestString, thx

File: curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md [hunk 2]
```diff
@@ -3,24 +3,24 @@ id: bad87fee1348bd9aedf08815
...
+  - text: 'Обязательно используйте процентное объявление <code>50%</code>.'
+    testString: 'assert(code.match(/50%/g), "Обязательно используйте процентное объявление <code>50%</code>.");'
```
Comment (greggubarev) [inline curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md]: Please turn back value of TestString

**Review by greggubarev — CHANGES_REQUESTED** [review]
Please turn back value of both TestString, thank you

File: curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md (hunk 1 above)
Comment (RandellDawson) [reply curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md]: @greggubarev The assert message is ignored anyway due to changes made recently.  What matters is the `text` above each `testString` which gets used instead.

File: curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md (hunk 2 above)
Comment (RandellDawson) [reply curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md]: @greggubarev The assert message argument is ignored anyway due to changes made recently.  What matters is the `text` above each `testString` which gets used instead.

Comment (RandellDawson) [reply curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md]: Also, technically, those added single quotes are not needed.  The above could be simply:
```yml
  - text: Обязательно используйте процентное объявление <code>50%</code>.
    testString: assert(code.match(/50%/g));
```

Comment (greggubarev) [reply curriculum/challenges/russian/01-responsive-web-design/basic-css/make-circular-images-with-a-border-radius.russian.md]: @RandellDawson thank you!

**Review by greggubarev — APPROVED** [review]

**Comment by greggubarev** [comment]
### LGTM 👍  Congrats on your first contribution! 
Image of Office
Thank you for the PR ✨ 🎉 We're happy to accept these changes and look forward to your future contributions.

Happy coding!

**Pull request merged**